Server/Interfaces/IChatServer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Server.Interfaces
+{
+    public interface IChatServer
+    {
+
+        List<IClient> clients { get; set; }
+        List<IRoom> rooms { get; set; }
+        Task StartAsync();
+        Task PrivateMessage(IClient client, string message);
```

Comment:
```suggestion
        Task SendPrivateMessage(IClient client, string message);
```
try to make methods look like actions